hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -688,6 +688,42 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       reseed: 15
     }
+
+    {
+      name: chip_sw_lc_ctrl_rma_to_scrap
+      uvm_test_seq: chip_sw_lc_ctrl_scrap_vseq
+      sw_images: ["//sw/device/tests/sim_dv:chip_sw_lc_ctrl_scrap_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+bypass_alert_ready_to_end_check=1",
+                 "+src_dec_state=DecLcStRma"]
+      reseed: 1
+    }
+    {
+      name: chip_sw_lc_ctrl_raw_to_scrap
+      uvm_test_seq: chip_sw_lc_ctrl_scrap_vseq
+      sw_images: ["//sw/device/tests/sim_dv:chip_sw_lc_ctrl_scrap_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+bypass_alert_ready_to_end_check=1",
+                 "+src_dec_state=DecLcStRaw"]
+      reseed: 1
+    }
+    {
+      name: chip_sw_lc_ctrl_test_locked0_to_scrap
+      uvm_test_seq: chip_sw_lc_ctrl_scrap_vseq
+      sw_images: ["//sw/device/tests/sim_dv:chip_sw_lc_ctrl_scrap_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+bypass_alert_ready_to_end_check=1",
+                 "+src_dec_state=DecLcStTestLocked0"]
+      reseed: 1
+    }
+    {
+      name: chip_sw_lc_ctrl_rand_to_scrap
+      uvm_test_seq: chip_sw_lc_ctrl_scrap_vseq
+      sw_images: ["//sw/device/tests/sim_dv:chip_sw_lc_ctrl_scrap_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+bypass_alert_ready_to_end_check=1"]
+      reseed: 3
+    }
```

Comment:
i think we can further improve this to have one test and use SV to randomize which starting state, but we can leave it as a TODO :)

Reply:
There is actually a randomized one here: `chip_sw_lc_ctrl_rand_to_scrap`
The other ones are just extra tests that make sure we always test some of the important transitions like RAW, RMA -> SCRAP.